everyvoice/base_cli/helpers.py

```diff
@@ -367,5 +369,9 @@ def train_base_command(
                 trainer.fit(model_obj, data, ckpt_path=tmp.name)
 
 
-def inference_base_command(_name: Enum):
-    pass
+def inference_base_command(
+    model: FastSpeech2,
+    config_args: List[str],
+):
+    config = model.config
+    update_config_from_cli_args(config_args, config)
```

Comment:
Doesn't this have to return the config? Just updating the config here makes the inference base command a little light - I would prefer to see some other things relevant to inference, but as I mentioned in the other comment, that can wait for a new issue.

Reply:
Yeah, we had discussions about it, and it's true it ended up quite light, but it's the thing we wanted to factor out from all the inference commands instead of adding it everywhere.